pkg/serverless/invocationlifecycle/init.go

```diff
@@ -212,15 +198,10 @@ func (lp *LifecycleProcessor) createWrappedEventBridgeSpan(eventBridgeEvent even
 }
 
 func (lp *LifecycleProcessor) initFromLambdaFunctionURLEvent(event events.LambdaFunctionURLRequest, region string, accountID string, functionName string) {
-	lp.requestHandler.event = event
 	if !lp.DetectLambdaLibrary() && lp.InferredSpansEnabled {
 		lp.GetInferredSpan().EnrichInferredSpanWithLambdaFunctionURLEvent(event)
 	}
 	lp.addTag(tagFunctionTriggerEventSource, functionURL)
 	lp.addTag(tagFunctionTriggerEventSourceArn, fmt.Sprintf("arn:aws:lambda:%v:%v:url:%v", region, accountID, functionName))
 	lp.addTags(trigger.GetTagsFromLambdaFunctionURLRequest(event))
 }
-
-func (lp *LifecycleProcessor) initFromStepFunctionPayload(event events.StepFunctionPayload) {
-	lp.requestHandler.event = event
```

Comment:
This value is never being used, the `Event() interface{}` was probably used by something before but not anymore